src/bundlers/node/lambdaHandler.ts

```diff
@@ -2,6 +2,9 @@
 /* eslint-disable no-useless-escape */
 
 export const lambdaHandler = (className: string): string => `
+/// This is an auto generated code. This code should not be modified since the file can be overwritten
+/// if new genezio commands are executed.
```

Comment:
```suggestion
/**
* This is an auto generated code. This code should not be modified since the file can be overwritten 
* if new genezio commands are executed.
*/
```